src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 public interface MachineProducer {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<Machine> get();
```

Comment:
The `MachineProducer` interface should be parameterized with a type parameter to ensure type safety. Modify the interface to use generics, such as `MachineProducer<T extends Machine>`, and update the `get()` method to return `List<T>`.